packages/core/plugin.schema.json

```diff
@@ -847,7 +849,7 @@
           }
         },
         "keyManagerDecryptJWE": {
-          "description": "Decrypts data",
+          "description": "Decrypts data This API may change without a BREAKING CHANGE notice.",
```

Comment:
In our case, `@beta` refers to the fact that semantic versioning may not be respected for changes related to those methods or classes.

Since this description may be used remotely through OpenAPI, I think it's fine if the beta warnings leak through.